browser/ui/views/speedreader/speedreader_bubble_util.cc

```diff
@@ -0,0 +1,69 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/browser/ui/views/speedreader/speedreader_bubble_util.h"
+
+#include <string>
+#include <vector>
+
+#include "ui/views/controls/styled_label.h"
+
+namespace {
+
+const std::vector<std::string> kReaderFontFamilies = {
```

Comment:
is is used at all? CC prohibits non-trivial global objects, so this should be an array or a fixed flat set

Reply:
not used. removed